classes/item.rb

```diff
@@ -0,0 +1,39 @@
+require 'securerandom'
+require_relative 'label'
+
+class Item
```

Comment:
Excellent work implementing almost all the requirements! 🥇 

- [ ] Kindly add a test for your Item class and all these methods as it's requested on requirements.